apps/chat_server/lib/chat_server/contexts/rooms/update_state.ex

```diff
@@ -25,14 +26,12 @@ defmodule ChatServer.UpdateState do
   defp update(params, state) do
     changeset = params
     |> Map.take([:id, :name])
-    |> Map.put(:status, state)
+    |> Map.put(:state, state)
 
     Schema.Room.update(params, changeset)
   end
 
   defp broadcast_update(room) do
-    event = %State{room: room}
-
-    ChatPubSub.broadcast! "events", "room:updated", event
+    BroadcastEvent.call("room:updated", room)
```

Comment:
Need to also add `alias ChatServer.BroadcastEvent` at the top of the context, otherwise this will throw an error

Reply:
Also, rename the file name from `update_state.ex` to `update_room.ex`.

Reply:
Note: both of these were handled in my merged PR against this branch.